src/pykeen/experiments/remixing.py

```diff
@@ -5,20 +5,24 @@
 import itertools as itt
 import json
 import logging
+import multiprocessing as mp
 import os
 import random
+from contextlib import nullcontext
+from functools import partial
 from typing import Type
 
 import click
 import matplotlib.pyplot as plt
 import seaborn as sns
 from scipy import stats
-from tqdm.auto import tqdm
+from tqdm import trange
```

Comment:
`tqdm.auto` ?

Reply:
it gets smart about if it's in a jupyter notebook or normal console

Reply:
oh shoot its supposed to be `tqdm.autonotebook`